python/masonite/config/application.py

```diff
@@ -0,0 +1,58 @@
+"""Application Settings."""
+
+import os
+
+from masonite import env
+
+"""Application Name
+This value is the name of your application. This value is used when the
+framework needs to place the application's name in a notification or
+any other location as required by the application or its packages.
+"""
+
+NAME = env('APP_NAME', 'Masonite 2.1')
+
+"""Application Debug Mode
+When your application is in debug mode, detailed error messages with
+stack traces will be shown on every error that occurs within your
+application. If disabled, a simple generic error page is shown
+"""
+
+DEBUG = env('APP_DEBUG', False)
```

Comment:
Just make sure debug is `False` when running this in production for a better speed boost

Reply:
I'm not sure about how environment variable going to work,  need to ask @waghanza

Reply:
by default, there is not
so having a default value to `False` is :ok:

Reply:
awesome 👌